tripleblindmarket/private-kit
# Update colors to match AA standards, consolidate/remove unused styles

#### Description:

This commit:
- Updates the color variables to meet WCAG 2.0 AA color accessibility standards. 
- Consolidates color variables that are very similar to each other after the change.
- Removes unused color variables from the codebase.
- Removes unused background image.
- Updates tests accordingly.

#### Linked issues:

https://trello.com/c/fJOPzesP/75-as-a-user-in-minnesota-with-accessibility-needs-i-want-to-ensure-that-all-color-contrasts-in-the-app-meet-wcag-20-aa-standards

#### Screenshots:


<img width="561" alt="Screen Shot 2020-07-16 at 9 34 33 AM" src="https://user-images.githubusercontent.com/2924479/87677309-9d118d00-c747-11ea-88fa-0e4512981155.png">
<img width="561" alt="Screen Shot 2020-07-16 at 9 33 52 AM" src="https://user-images.githubusercontent.com/2924479/87677316-9e42ba00-c747-11ea-8aba-bcf405b9b546.png">
<img width="561" alt="Screen Shot 2020-07-16 at 9 34 39 AM" src="https://user-images.githubusercontent.com/2924479/87677590-f24d9e80-c747-11ea-8ae7-36b2c3c3e517.png">


#### How to test:

- Run the app locally.
- Ensure the UI matches the attached screenshots.
- Review the full UI to ensure that all colors and gradients are rendering properly and readable.
